v2/client/src/components/pages/survey/SurveyQs.js

```diff
@@ -165,14 +170,14 @@ const renderQuestionInputType = (
             allowClear={false}
             count={6}
             onChange={value => {
-              handleStarChange(value, questionId);
+              handleStarChange(value, questionId, participantField);
               return setCurrentQuestion(nextQuestionID);
             }}
             unanswered={errorArray.includes(questionId) && !answers[questionId]}
-            data-group={group}
+            data-group={group.text}
             data-field={participantField}
             onBlur={() => setCurrentQuestion(nextQuestionID)}
-            value={answers[questionId] && answers[questionId].answer + 1}
+            value={answers[questionId] && answers[questionId].answer / 2 + 1}
```

Comment:
I think it would be easier to deal with answers as it is. then convert on the formula.